definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-prod.json

```diff
@@ -1 +1,9 @@
-[]
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LabelSolStatementOTruthPara-4",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "CRU"
```

Comment:
if this has the same value for prod and non prod why it's not in `core` file?

Reply:
It was added to a prod file, but i ended up removing it. It will only be on a feature toggle so it doesnt need to be here at all